src/waveform/renderers/waveformmark.h

```diff
@@ -1,39 +1,35 @@
 #ifndef WAVEFORMMARK_H
 #define WAVEFORMMARK_H
 
-#include <QString>
+#include <QDomNode>
 #include <QImage>
-#include <QColor>
 
-#include "preferences/usersettings.h"
-#include "skin/skincontext.h"
+#include "util/memory.h"
+
+#include "waveform/renderers/waveformmarkproperties.h"
 
 class ControlProxy;
-class QDomNode;
+class SkinContext;
 class WaveformSignalColors;
 
 class WaveformMark {
   public:
     WaveformMark();
```

Comment:
The default constructor has become obsolete now, because C++ generates one for you. If you want to keep it for documentation purposes then use
WaveformMark() = default;
and delete the implementation. I like C++1x ;)